client/src/app/App.js

```diff
@@ -1300,6 +1300,8 @@ export class App extends PureComponent {
 
   async saveTab(tab, options) {
 
+    this.triggerAction('saveEditing');
```

Comment:
If we decide to go this direction, we will have to make sure, all kind of Editors support this. E.g. it would currently throw an error inside a DMN file:

```js
saveEditing is not a registered action
    at error (webpack-internal:///./node_modules/dmn-js-drd/node_modules/diagram-js/lib/features/editor-actions/EditorActions.js:260:10)
    at DrdEditorActions.EditorActions.trigger (webpack-internal:///./node_modules/dmn-js-drd/node_modules/diagram-js/lib/features/editor-actions/EditorActions.js:170:11)
    at DmnEditor._defineProperty (webpack-internal:///./src/app/tabs/dmn/DmnEditor.js:344:54)
    at MultiSheetTab._defineProperty (webpack-internal:///./src/app/tabs/MultiSheetTab.js:139:21)
    at EditorTab.triggerAction (webpack-internal:///./src/app/tabs/EditorTab.js:36:34)
    at App.failSafe (webpack-internal:///./src/app/App.js:657:18)
    at App.eval (webpack-internal:///./src/app/App.js:1856:20)
    at App.saveTab (webpack-internal:///./src/app/App.js:1390:10)
    at App.failSafe (webpack-internal:///./src/app/App.js:564:21)
    at App.eval (webpack-internal:///./src/app/App.js:1856:20) [ error ]
This error may be the result of a plug-in compatibility issue. [ info ]
Disable plug-ins (restarts the app) [ info ]
```

Reply:
I closed the bpmn-js PR and implemented this as a module under the Camunda modeler within this PR, so this is fixed. Thanks for the catch!